README.md

```diff
@@ -127,6 +127,7 @@ To execute command within chroot environment you should use chroot communicator:
  }
 ]
 ```
+For an extended example that does provisioning via script including provisioning of a resize-root-fs systemd service see [here](./boards/raspberry-pi/archlinuxarm.json).
```

Comment:
I would rewrite this a bit :) WDYT?

```
This plugin doesn't resize partitions on the base image. However, you can easily expand partition size at the boot time with a systemd service. [Here](./boards/raspberry-pi/archlinuxarm.json) you can find real-life example, where a raspberry pi root-fs partition expands to all available space on sdcard.
```

Reply:
This plugin doesn't resize partitions on the base image. However, you can easily expand partition size at the boot time with a systemd service. [Here](./boards/raspberry-pi/archlinuxarm.json) you can find real-life example, where a raspberry pi root-fs partition expands to all available space on sdcard.